CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.1.0] - 2023-10-15
```

Comment:
To avoid the circularity of having to release I would typically not detailed the changes in the `CHANGELOG.md` until after a release has been made.

We have automatic releasing to PyPI set up through a GitHub Action (`.github/workflows/release-to-pypi.yaml`) such that when a release is made, which obliges a tag to be added to the current `HEAD` commit, it triggers building and publishing. 

There is also `.github/workflows/release-to-orda.yaml` which is triggered releases and tagging commits which uploads the artefacts to Sheffields ORDA figshare site (although it often times out and needs re-running).

I'll see if there is a way of automatically building the `CHANGELOG.md` from Release Notes in some manner (maybe there is an action that can be included).

Reply:
I think adding changes to the changelog under the "Unreleased" section seems like a nice thing to do in a PR, but I'll agree that pushing a release like this in a PR might be a bit much!